integration/transition-state-test.ts

```diff
@@ -254,9 +256,11 @@ test.describe("rendering", () => {
           search: "?redirected",
           hash: "",
           state: {
-            isRedirect: true,
-            setCookie: false,
-            type: "loader",
+            _isRedirect: true,
+            // These were private API for transition manager that are no longer
+            // needed with the new router so OK to disappear
+            // setCookie: false,
+            // type: "loader",
```

Comment:
Passing integration tests from `dev` change slightly since we use new private API on location state to detect `type`